### ISSUE TITLE
humbug_get_contents() is deprecated

### ISSUE THREAD

**Issue by waltertamboer** [humbug/phar-updater#45]
```
  [Humbug\SelfUpdate\Exception\HttpRequestException]                                                                
  humbug_get_contents() is deprecated since 1.1.0 and will be removed in 4.0.0. Use Humbug/get_contents() instead.
```

**Comment by theofidry** [comment]
`4.0.0`, not `2.0.0`?

**Comment by theofidry** [comment]
Meh, fuck me, needs to be fixed

**Comment by theofidry** [comment]
Closed via #46
